### PR TITLE
Containerization changes

### PR DESCRIPTION
**Pull request by Akreddy123**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/ContinuousAssurance/ContinuousAssurance.ps1 [hunk 1]
```diff
@@ -165,6 +171,11 @@ function Install-AzSKContinuousAssurance
 		[Alias("stsc")]
 		$SkipTargetSubscriptionConfig,
 
+		[Parameter(Mandatory = $true, ParameterSetName = "ContainerMode", HelpMessage="Switch to use container instead of automation account for AzSK CA.")]
+		[switch]
+		[Alias("uc")]
+		$UseContainers,
```
Comment (v-zhgup) [inline src/AzSK/ContinuousAssurance/ContinuousAssurance.ps1]: Should we use singular container if we are spinning up a single container instance

File: src/AzSK/Framework/Core/CAwithACI/ContinuousAssurance.ps1 [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+		}
+        if(($functionAppObjet | Measure-Object).Count -le 0)
+        {
+            New-AzResource -ResourceType 'Microsoft.Web/Sites' -ResourceName $this.FunctionAppName -kind 'functionapp' -Location $this.CALocation -ResourceGroupName $this.ResourceGroup -Properties @{} -Force;
+        }
+        $AzSKFunctionApp = Set-AzWebApp -Name $this.FunctionAppName -ResourceGroupName $this.ResourceGroup -AppSettings $AppSettings -AssignIdentity $true -HttpsOnly $true								
+        Start-Sleep -Seconds 10
+        $this.FunctionAppMSI = $AzSKFunctionApp.Identity.PrincipalId
+        #TBD
+        #$zipFilePath = "C:\Users\AKALETI\Desktop\mclass\AzSK-Containerization\temp.zip";
```
Comment (Akreddy123) [inline src/AzSK/Framework/Core/CAwithACI/ContinuousAssurance.ps1]: remove comment

File: src/AzSK/ContinuousAssurance/ContinuousAssurance.ps1 [hunk 3]
```diff
@@ -58,6 +58,7 @@ function Install-AzSKContinuousAssurance
 	Param(
 		[Parameter(Position = 0, Mandatory = $true, ParameterSetName = "Default", HelpMessage="Id of the subscription in which Automation Account needs to be installed.")]
 		[Parameter(Position = 0, Mandatory = $true, ParameterSetName = "CentralScanMode", HelpMessage="Id of the subscription in which Automation Account needs to be installed.")]
+		[Parameter(Position = 0, Mandatory = $true, ParameterSetName = "ContainerMode", HelpMessage="Id of the subscription in which Automation Account needs to be installed.")]
```
Comment (VishalHaibatpure) [inline src/AzSK/ContinuousAssurance/ContinuousAssurance.ps1]: Normal ICA/UCA parameterset needs to be tested thoroughly

Comment (Akreddy123) [reply src/AzSK/ContinuousAssurance/ContinuousAssurance.ps1]: yes

**Pull request merged**